### ISSUE TITLE
Project doesn't compile and give 6 errors

### ISSUE THREAD

**Issue by JimClermonts** [VMadalin/kotlin-sample-app#8]
## Description

Downloaded the latest Master branch and got a marvel API key. Then I was prompted to update to Android Studio 4.0 which is still in early phases. After that I got 6 compile errors and tried to downgrade gradle. This project is not compiling!

## Basic Information

-   Device type: Mac
-   OS version: OSX Catalina
-   App version: 

## Details
https://imgshare.io/images/2020/01/05/123.png

**Comment by VMadalin** [comment]
Thank you so much @JimClermonts for the reported issue. I'm absolutely agree with you Android Studio 4.0 it's still in early phases. I tried to add it for thw full support for `gradle-dsl` but after update it locally I don't have any problem. 

After check your issue I tried to clone again the repo and at that moment I see the issue. For the moment I rollback to the latest stable version Android 3.5.3 waiting until the version 4.0 will become more stable.
